projects/go-lib/src/lib/components/go-accordion/go-accordion-panel.component.scss

```diff
@@ -58,19 +58,19 @@
   position: relative;
 
   &:hover {
-    background-color: $theme-light-bg-hover;
+    background-color: darken($base-light, 7%);
   }
 
   &.go-accordion-panel__header--dark:hover {
-    background-color: $theme-dark-bg-hover;
+    background-color: darken($base-dark-secondary, 3.5%);
   }
 }
 
 .go-accordion-panel__header--active {
-  background-color: $theme-light-bg-active;
+  background-color:  darken($base-light, 3.5%);
 
   &.go-accordion-panel__header--dark {
-    background-color: $theme-dark-bg-active;
+    background-color: darken($base-dark-secondary, 7%);
```

Comment:
I'm wondering why these needed to change? This seems to break the pattern of the hover state being lighter than the active state established by the changes below to the variable values.

Reply:
@jaredami it does break the pattern... but this is what UX wanted to go with :shrug: